src/Microsoft.AspNet.IISPlatformHandler/IISPlatformHandlerMiddleware.cs

```diff
@@ -35,10 +44,20 @@ public IISPlatformHandlerMiddleware(RequestDelegate next, IOptions<IISPlatformHa
 
             _next = next;
             _options = options.Value;
+            _logger = loggerFactory.CreateLogger<IISPlatformHandlerMiddleware>();
         }
 
         public async Task Invoke(HttpContext httpContext)
         {
+            if (_options.FlowClientCertificate)
+            {
+                var header = httpContext.Request.Headers[MSPlatformHandlerClientCert];
+                if (!StringValues.IsNullOrEmpty(header))
+                {
+                    httpContext.Features.Set<ITlsConnectionFeature>(new ForwardedTlsConnectionFeature(_logger, header));
```

Comment:
Do we want to create logger with "ForwardedTlsConnectionFeature" category instead of using "IISPlatformHandlerMiddleware" one?